src/prted/pmix/pmix_server_dyn.c

```diff
@@ -549,9 +551,11 @@ static void interim(int sd, short args, void *cbdata)
             pmix_server_cache_job_info(jdata, info);
 
         } else if (PMIX_CHECK_KEY(info, PMIX_DEBUG_STOP_IN_APP)) {
+            rank = PMIX_RANK_WILDCARD;
             prte_set_attribute(&jdata->attributes, PRTE_JOB_STOP_IN_APP, PRTE_ATTR_GLOBAL,
-                               info->value.data.string, PMIX_STRING);
+                              &rank, PMIX_PROC_RANK);
             /* also must add to job-level cache */
+            PMIX_INFO_LOAD(info, PMIX_DEBUG_STOP_IN_APP, &rank, PMIX_PROC_RANK);
```

Comment:
You really don't want to do this - you are overwriting what we are being told and losing the label information that the app requires.